patches/net/minecraft/world/item/crafting/RecipeMap.java.patch

```diff
@@ -0,0 +1,31 @@
+--- a/net/minecraft/world/item/crafting/RecipeMap.java
++++ b/net/minecraft/world/item/crafting/RecipeMap.java
+@@ -13,7 +_,7 @@
+ 
+ public class RecipeMap {
+     public static final RecipeMap EMPTY = new RecipeMap(ImmutableMultimap.of(), Map.of());
+-    private final Multimap<RecipeType<?>, RecipeHolder<?>> byType;
++    private Multimap<RecipeType<?>, RecipeHolder<?>> byType;
+     private final Map<ResourceKey<Recipe<?>>, RecipeHolder<?>> byKey;
+ 
+     private RecipeMap(Multimap<RecipeType<?>, RecipeHolder<?>> p_379497_, Map<ResourceKey<Recipe<?>>, RecipeHolder<?>> p_380280_) {
+@@ -31,6 +_,19 @@
+         }
+ 
+         return new RecipeMap(builder.build(), builder1.build());
++    }
++
++    public void order() {
++        Map<ResourceKey<Recipe<?>>, Integer> recipePriorities = net.neoforged.neoforge.common.NeoForgeEventHandler.getRecipePriorityManager().getRecipePriorities(); // Neo: get overriding recipe priorities
```

Comment:
I'm not a fan of accessing other reload listeners statically like that. What I would maybe do is adding a `setPriorityMap(Map<...> priorities)` to `RecipeManager`, then calling it from the `apply` method of the `RecipePriorityManager`.